src/app/Dashboard/components/LogPointsModal.js

```diff
@@ -0,0 +1,168 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import TextField from '@material-ui/core/TextField';
+import StartIcon from '@material-ui/icons/Star';
+import { MenuItem, ButtonComponent } from '../../common/components';
+import actions from '../operations/actions';
+
+/**
+   * @name LogActivityForm
+   * @summary Returns Form
+   * @returns Returns a form
+   */
+export class LogActivityForm extends Component {
+  state = {
+    categoryOption: [],
+    numberOfParticipants: '',
+    description: '',
+    activityDate: '',
+  };
+
+  static defaultProps = {
+    categories: [],
+    show: false,
+    logActivity: () => {},
+    close: () => {},
+  }
+
+  static propTypes = {
+    show: PropTypes.bool,
+    logActivity: PropTypes.func,
+    categories: PropTypes.arrayOf(PropTypes.shape({})),
+    close: PropTypes.func,
+  };
+
+  handleChange = prop => (event) => {
```

Comment:
It's difficult for someone to know what this prop is

Reply:
@Kachulio1 I have included a summary that explains what it does.